tools/harness-simulation/harness/Web/data/updateDeviceFields.py

```diff
@@ -0,0 +1,26 @@
+import os
+import sys
+import xml.etree.ElementTree as ET
+
+HARNESS_XML_PATH = r'%s\GRL\Thread1.2\Web\data\deviceInputFields.xml' % os.environ['systemdrive']
```

Comment:
Maybe we should also make this a command line argument. For example, do we need to switch to `Thread 1.3` in the future?

Reply:
The path is fixed. It's always Thread1.2 for all 1.1/1.2/1.3 certifications.